src/config.ts

```diff
@@ -51,6 +51,18 @@ export const WECHATY_PUPPET_SERVICE_ENDPOINT = () => {
   return undefined
 }
 
+export const GET_CHATIE_ENDPOINT_LIST = () => {
```

Comment:
I think we should not use a list here anymore.

When we add this **list** to our code, the reasons are: some server has issues visiting the Cloudflare/Heroku IP (slow or even timeout).

After we can override this endpoint, then the user who has the above issue will be able to use their private endpoint to solve the problem.

So I'd like to suggest that we only keep one endpoint here.

Please let me know what you think @windmemory @su-chang

Reply:
The default endpoint still keep `[ 'https://api.chatie.io', 'http://68.79.16.140' ]`, and keep only one endpoint is for private endpoint?

Reply:
I'm considering only keep one default endpoint, but it's not decided yet.

Reply:
Vote for one default endpoint.

Multiple endpoints was introduced to resolve network latency issue, and seems like we don't have that problem anymore, so we can change it back to single endpoint.